networkit/cpp/io/ThrillGraphBinaryWriter.cpp

```diff
@@ -0,0 +1,115 @@
+#include "ThrillGraphBinaryWriter.h"
+
+namespace {
+    //! Append a varint to the writer.
+template <typename stream>
+stream& PutVarint(stream& s, uint64_t v) {
```

Comment:
Uh. Manually unrolling this loop certainly is ugly. Are compilers not smart enough to optimize the following?
```
if(!v)
   s << uint8_t(0);
while(v) {
    auto u = v & 0x7F;
    v >>= 7;
    s << uint8_t(u | (v ? 0x80 : 0));
}
```

Reply:
Fixed in a7ff6f55733400afb172d0a24669715cb7fea8d2.